src/sentry/static/sentry/app/components/activity/note/types.tsx

```diff
@@ -20,10 +22,12 @@ export type Mentioned = [string, string];
  */
 export type MentionChangeEvent = Parameters<OnChangeHandlerFunc>[0];
 
-export type CreateError = {
-  detail: {
-    message: string;
-    code: number;
-    extra: any;
-  };
-};
+export type CreateError =
```

Comment:
everywhere we use this type, we also use `null`, like` CreateError | null`. So I was wondering if we should also add null in this type 🧐  but I am only thinking out loud... this is something for another PR 😉

Reply:
I thought about that too, that's more practical for sure but having it like this in the components' props is ver explicit and helps understand how this prop behaves.